api/controllers/likeController.js

```diff
@@ -0,0 +1,44 @@
+const { likeService } = require('../services');
+const { CustomError } = require('../utils/error');
+
+const createLike = async (req, res) => {
+    try {
+        const { productId } = req.params;
+        const userId = req.user;
+
+        const data = await likeService.createLike(userId, productId);
+
+        const result = {
+            ...data,
+            is_liked: true,
+        };
+
+        return res.status(201).json({ result: result });
```

Comment:
```suggestion
        return res.status(201).json({ result });
```
이러한 경우에는 suggestion 같이 작성하여도 똑같이 출력됩니다.